src/Stacktrace.php

```diff
@@ -134,14 +135,26 @@ public function __construct(Configuration $config)
     /**
      * Get the array representation.
      *
-     * @return array
+     * @return array[]
      */
     public function toArray()
     {
         return $this->frames;
     }
 
     /**
+     * Get the stacktrace frames.
+     *
+     * This is the same as calling toArray.
```

Comment:
thought it would be nice to clarify they were the same from the outside perspective :)